src/Config/packages.aget

```diff
@@ -2,12 +2,12 @@
   "nuget": {
     "references": {
       "DynamicLanguageRuntime" : "1.2.1",
-      "DynamoVisualProgramming.Core" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.DynamoCoreNodes" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.DynamoServices" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.Tests" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.WpfUILibrary" : "2.1.0-beta7020",
-      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.1.0-beta7020",
+      "DynamoVisualProgramming.Core" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.DynamoCoreNodes" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.DynamoServices" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.Tests" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.WpfUILibrary" : "2.1.0-beta7321",
+      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.1.0-beta7321",
```

Comment:
Are these based on the latest NuGet versions posted? Does this file need to be updated manually each time you post new NuGet's?

Reply:
@aparajit-pratap Yes, unfortunately Revit team changed from always consuming latest beta to fixed version to slow down the process and making sure the build is stable